### PR TITLE
chore: update test image for uta

### PR DESCRIPTION
**Pull request by korikuzma**
Part of #312

### CODE REVIEW

**Review by jsstevenson — APPROVED** [review]
good call

File: docker-compose.yml [hunk 1]
```diff
@@ -17,12 +17,12 @@ services:
...
     ports:
       - 5000:5000
-  
+
   uta:
     # Test:
-    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
-    # 249909
-    image: biocommons/uta:uta_20180821
+    # psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from uta_20210129b.transcript'
```
Comment (larrybabb) [inline docker-compose.yml:23]: I think the corresponding documentation on the README.md should be changed as well (this took me a minute when I was trying to use the install instructions to figure out.

```
snafu$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
249909
```

**Review by larrybabb — COMMENTED** [review]
one README.md change to sync.

File: docker-compose.yml (hunk 1 above)
Comment (larrybabb) [reply docker-compose.yml:23]: i guess I missed the review window ;)

Comment (korikuzma) [reply docker-compose.yml:23]: Good catch!!

**Pull request merged**